File: seed/csharp-sdk/path-parameters/no-custom-config/src/SeedApi/Organizations/Requests/OrganizationsSearchOrganizationsRequest.cs
using global::System.Text.Json.Serialization;
using SeedApi.Core;

namespace SeedApi;

[Serializable]
public record OrganizationsSearchOrganizationsRequest
{
    [JsonIgnore]
    public required string TenantId { get; set; }

    [JsonIgnore]
    public required string OrganizationId { get; set; }

    [JsonIgnore]
    public int? Limit { get; set; }

    /// <inheritdoc />
    public override string ToString()
    {
        return JsonUtils.Serialize(this);
    }
}

File: seed/php-sdk/path-parameters/inline-path-parameters/src/Organizations/Requests/OrganizationsSearchOrganizationsRequest.php
<?php

namespace Seed\Organizations\Requests;

use Seed\Core\Json\JsonSerializableType;

class OrganizationsSearchOrganizationsRequest extends JsonSerializableType
{
    /**
     * @var string $tenantId
     */
    public string $tenantId;

    /**
     * @var string $organizationId
     */
    public string $organizationId;

    /**
     * @var ?int $limit
     */
    public ?int $limit;

    /**
     * @param array{
     *   tenantId: string,
     *   organizationId: string,
     *   limit?: ?int,
     * } $values
     */
    public function __construct(
        array $values,
    ) {
        $this->tenantId = $values['tenantId'];
        $this->organizationId = $values['organizationId'];
        $this->limit = $values['limit'] ?? null;
    }
}
